[2025-02-15T00:56:18.628+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data_ETL.get_location_dict manual__2025-02-15T00:56:13.638531+00:00 [queued]>
[2025-02-15T00:56:18.640+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data_ETL.get_location_dict manual__2025-02-15T00:56:13.638531+00:00 [queued]>
[2025-02-15T00:56:18.641+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 1
[2025-02-15T00:56:18.657+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): get_location_dict> on 2025-02-15 00:56:13.638531+00:00
[2025-02-15T00:56:18.663+0000] {standard_task_runner.py:57} INFO - Started process 1708 to run task
[2025-02-15T00:56:18.669+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'weather_data_ETL', 'get_location_dict', 'manual__2025-02-15T00:56:13.638531+00:00', '--job-id', '65', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmp8mzhpblo']
[2025-02-15T00:56:18.673+0000] {standard_task_runner.py:85} INFO - Job 65: Subtask get_location_dict
[2025-02-15T00:56:18.784+0000] {task_command.py:410} INFO - Running <TaskInstance: weather_data_ETL.get_location_dict manual__2025-02-15T00:56:13.638531+00:00 [running]> on host 2079ceed134c
[2025-02-15T00:56:18.969+0000] {taskinstance.py:1568} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airscholar' AIRFLOW_CTX_DAG_ID='weather_data_ETL' AIRFLOW_CTX_TASK_ID='get_location_dict' AIRFLOW_CTX_EXECUTION_DATE='2025-02-15T00:56:13.638531+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-15T00:56:13.638531+00:00'
[2025-02-15T00:56:19.010+0000] {python.py:183} INFO - Done. Returned value was: None
[2025-02-15T00:56:19.030+0000] {taskinstance.py:1368} INFO - Marking task as SUCCESS. dag_id=weather_data_ETL, task_id=get_location_dict, execution_date=20250215T005613, start_date=20250215T005618, end_date=20250215T005619
[2025-02-15T00:56:19.086+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2025-02-15T00:56:19.139+0000] {taskinstance.py:2674} INFO - 1 downstream tasks scheduled from follow-on schedule check
